image/ElaineSchenk/Contrast_Test_Script.py

```diff
@@ -0,0 +1,57 @@
+#get image to actually perform alg on

```

Comment:
Header Block would also include authorship, license, copyright, dates and description of what the driver/module does.